### PR TITLE
	Add global error handler to always return json

### PR DESCRIPTION
**Pull request by mbryzek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,20 @@
...
 
     play.modules.enabled += "io.flow.play.clients.ConfigModule"
     play.modules.enabled += "io.flow.play.clients.RegistryModule"
     play.modules.enabled += "io.flow.play.clients.TokenModule"
     play.modules.enabled += "io.flow.play.clients.UserModule"
 
-# Traits
+## Global error handler for JSON APIs
+
+This error handler capture client and server errors, returning
```
Comment (chrhicks) [inline README.md]: ...returning ?

**Comment by ericluria** [comment]
LGTM (and Roth)

**Comment by chrhicks** [comment]
lgtm!

File: README.md (hunk 1 above)
Comment (mbryzek) [inline README.md]: updated

**Pull request merged**